packages/gatsby-remark-images-contentful/src/utils/index.js

```diff
@@ -1,20 +1,34 @@
+const { URL } = require(`url`)
 const axios = require(`axios`)
 
-const getBase64Img = async url => {
-  const response = await axios({
-    method: `GET`,
-    responseType: `arraybuffer`,
-    url: `${url}`,
-  })
+// This should be replaced with the solution for https://github.com/gatsbyjs/gatsby/issues/24220
+const getBase64Img = async (url, reporter) => {
+  try {
+    const response = await axios({
+      method: `GET`,
+      responseType: `arraybuffer`,
+      url: `${url}`,
+    })
 
-  const base64Img = `data:${
-    response.headers[`content-type`]
-  };base64,${new Buffer(response.data).toString(`base64`)}`
+    const base64Img = `data:${
+      response.headers[`content-type`]
+    };base64,${new Buffer(response.data).toString(`base64`)}`
 
-  return base64Img
+    return base64Img
+  } catch (err) {
+    reporter.panic(`Failed downloading the base64 image for ${url}`, err)
```

Comment:
I'm pretty sure this will stop the process, or at least that the process shouldn't be considered to continue after a `.panic` is called. The next line implies it might want to recover regardless. Just make sure that you intended to call .panic() here and not a non-fatal `.error` or `.warn`...

Reply:
Hmm, IMO it should fail if it could not generate the preview.